platform_service_accounts_api/schema.py

```diff
@@ -31,8 +32,18 @@ async def _wrapped(self: Any, request: aiohttp.web.Request) -> Any:
     return _decorator
 
 
+NAME_PATTERN = re.compile(
+    r"[a-z0-9](?:[-a-z0-9]*[a-z0-9])?(?:\.[a-z0-9](?:[-a-z0-9]*[a-z0-9])?)*"
+)
+
+
+def validate_name(name: str) -> None:
```

Comment:
Minor: I guess you can use [marshmallow.validate.Regexp](https://marshmallow.readthedocs.io/en/stable/marshmallow.validate.html#marshmallow.validate.Regexp) here, something like:
```
name = fields.String(required=False, missing=None, validate=validate.Regexp(NAME_PATTERN)
```

Reply:
Yes, I considered this alternative. But with custom function we control the error message and can easily add additional restrictions (like forbid some reserved names or names consisting only of digits and dots).